src/Microsoft.Azure.CosmosRepository.AspNetCore/Extensions/HealthChecksBuilderExtensions.cs

```diff
@@ -55,4 +55,32 @@ public static class HealthChecksBuilderExtensions
 
         return builder;
     }
+
+    /// <summary>
+    /// Add a health check for Azure Cosmos DB by registering <see cref="AzureCosmosDbHealthCheck"/> for given <paramref name="builder"/>.
```

Comment:
I think this looks good, just need to edit the two summary XML tags so it's clear, that one automatically scans for all item types and the other allows you to manually specify

Reply:
Sorted 👍